packages/react/src/date-picker/DateRangePicker.test.tsx

```diff
@@ -269,7 +269,7 @@ describe('DateRangePicker', () => {
 			required: false,
 		});
 		expect(await await (await getLegend(defaultLegend)).textContent).toEqual(
-			`${defaultLegend}(optional)`
```

Comment:
These tests always felt a bit weird because there was no space between the primary and secondary label. I'm very happy to see this change :D